cmd/repo-brancher/main.go

```diff
@@ -151,66 +147,39 @@ func main() {
 
 			// when we're initializing the branch, we just want to make sure
 			// it is in sync with the current branch that is promoting
-			branchLogger := logger.WithField("future-branch", futureBranch)
+			logger := logger.WithField("future-branch", futureBranch)
 			command := []string{"ls-remote", remote.String(), fmt.Sprintf("refs/heads/%s", futureBranch)}
-			cmdLogger := branchLogger.WithFields(logrus.Fields{"commands": fmt.Sprintf("git %s", strings.Join(command, " "))})
-			cmd := exec.Command("git", command...)
-			cmd.Dir = repoDir
-			cmdLogger.Debug("Running command.")
-			if out, err := cmd.CombinedOutput(); err != nil {
-				cmdLogger.WithError(err).WithFields(logrus.Fields{"output": string(out)}).Error("Failed to execute command.")
+			if err := executeGitCmd(logger, command...); err != nil {
 				failed = true
 				continue
-			} else {
-				cmdLogger.WithFields(logrus.Fields{"output": string(out)}).Debug("Executed command.")
-				if string(out) == "" && !o.fastForward {
```

Comment:
This drops the only `fastForward` consumer that I can find.  Should we deprecate `--fast-forward`, drop it from openshift/release, and then drop the command-line option here?